kuma/javascript/src/payments/routes.jsx

```diff
@@ -0,0 +1,71 @@
+// @flow
+import * as React from 'react';
+import Route from '../route.js';
+import Page from './pages/page.jsx';
+import LandingPage from './pages/index.jsx';
+import ThankYouPage from './pages/thank-you.jsx';
+import TermsPage from './pages/terms.jsx';
```

Comment:
This is really cool, love it. Thanks for doing this @mindy and @peterbe for suggesting we do this. 🍵